grassroots-backend/src/App.controller.ts

```diff
@@ -1,13 +1,41 @@
-import { Controller, Get } from "@nestjs/common";
+import { Controller, Get, Post, UseGuards, Request } from "@nestjs/common";
 import { AppService } from "./App.service";
 import { HelloOutDTO } from "./app/entities/Hello.dto";
+import { UserEntity } from "./grassroots-shared/User.entity";
+import { DefaultAuthGuard } from "./auth/DefaultAuth.guard";
+import type { GrassrootsRequest } from "./types/GrassrootsRequest";
+import { ApiBody } from "@nestjs/swagger";
+import { AuthService } from "./auth/Auth.service";
 
 @Controller()
 export class AppController {
-  constructor(private readonly appService: AppService) {}
+  constructor(
+    private readonly appService: AppService,
+    private authService: AuthService,
+  ) {}
 
   @Get()
   getHello(): HelloOutDTO {
     return this.appService.getHello();
   }
+
+  @UseGuards(DefaultAuthGuard)
```

Comment:
The docs put this in the AppController, but I'm tempted to move it to an AuthController. Any thoughts?

Reply:
I'll move it!